src/app/cli/src/init/client.ml

```diff
@@ -314,10 +358,19 @@ let verify_receipt =
       ~doc:"TOKEN_ID The token ID for the account"
       (optional_with_default Token_id.default Cli_lib.Arg_type.token_id)
   in
+  let config_file = Cli_lib.Flag.conf_file in
   Command.async ~summary:"Verify a receipt of a sent payment"
     (Cli_lib.Background_daemon.rpc_init
-       (Args.zip4 payment_path_flag proof_path_flag address_flag token_flag)
-       ~f:(fun port (payment_path, proof_path, pk, token_id) ->
+       (Args.zip5 config_file payment_path_flag proof_path_flag address_flag
+          token_flag )
+       ~f:(fun port (config_file, payment_path, proof_path, pk, token_id) ->
+         let%bind compile_config =
```

Comment:
Code below is repeated more than ten times, probably we could remove the duplication.